app/controllers/api_controller.rb

```diff
@@ -16,8 +17,24 @@ def current_board
     end
   end
 
+  def opponent_board
+    if current_player == current_game.player_1
+      :player_2_board
+    elsif current_player == current_game.player_2
+      :player_1_board
+    end
+  end
+
   def current_game
     @game ||= Game.find(params[:game_id])
   end
 
+  private
+
+  def require_api_key
+    user = User.find_by_api_key(request.headers['X-API-Key'])
+
+    render json: {message: 'Unauthorized'}.to_json, status: 401 and return false if user.nil?
```

Comment:
@jmejia @icorson3 Is this a good way to handle this? To scope the entire `/api/v1/` namespace to require valid API keys? Or is there some other method we should look into? 

This is already a merged PR but it can always be fixed in another branch.